src/gamefalloutnv.cpp

```diff
@@ -90,7 +90,7 @@ QString GameFalloutNV::identifyGamePath() const
       auto subDirs   = startPath.entryList({"Fallout New Vegas*"},
                                            QDir::Dirs | QDir::NoDotAndDotDot);
       if (!subDirs.isEmpty())
-        result = subDirs.first();
+        result = startPath.absolutePath() + QDir::separator() + subDirs.first();
```

Comment:
You could use `startPath.absoluteFilePath(subDirs.first())`.

Reply:
Yep, you're right.